modules/nf-core/iqtree/tests/iqtree_basic.config

```diff
@@ -1,5 +1,5 @@
 process {
     withName: "IQTREE" {
-        ext.args = "-bb 1000 -wbt"
+        ext.args = "-m LG -wt"
```

Comment:
If any of these options are mandatory to make the tool run with the minimum number of parameters then these should all be supplied as `input:`

Reply:
Neither of them are mandatory. With the first option I'm supplying a model to skip a model selection process (which I reasoned would unnecessarily slow down tests).
The other one is a flag required for a specific output type which would not work in any of the other tests. I though it would be easiest to include it here, but I can just move it to a separate test and keep the basic test truly basic.